airgun/browser.py

```diff
@@ -298,6 +261,58 @@ def _get_docker_browser(self):
         self._webdriver = self._docker.webdriver
         return self._webdriver
 
+    def _get_remote_browser(self):
+        """Returns remote webdriver instance of selected ``browser``.
+
+        Note: should not be called directly, use :meth:`get_browser` instead.
+        """
+        self._webdriver = webdriver.Remote(
+            command_executor=settings.selenium.command_executor,
```

Comment:
What will happen if we pass `None` here (which is the default)? Will it throw exception with meaningful message?
Perhaps we should be a little more defensive, and first ensure that this variable is set?

Reply:
When command_executor is None,  webdriver.Remote will try to use ''http://127.0.0.1:4444/wd/hub" as the default command_executor to connect, if the default command_executor is not available, exception message will be raised:
`urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5a7ff01080>: Failed to establish a new connection: [Errno 111] Connection refused',))`

I think the exception message is fair, webdriver.Remote has the ability to verify the command_executor's value.

Reply:
Agreed, this message looks good. I think people should be able to understand where the problem is after seeing it.
Thanks for checking this.